wagtail/documents/wagtail_hooks.py

```diff
@@ -72,7 +73,7 @@ def editor_js():
             window.chooserUrls.documentChooser = '{0}';
         </script>
         """,
-        reverse("wagtaildocs:chooser"),
+        reverse("wagtaildocs_chooser:choose"),
```

Comment:
Looking at this, it may be worth adding upgrade consideration notes for anyone either using the direct chooser url in own code, or tests.

Reply:
Have added a note in d588b79ac1ba6f5fadf3f00d1edd29243a4255e2, which also covers the more extensive changes to URL routes in the snippets app (@laymonage, please double-check that I've got the details right...). I'm in two minds about whether it's worth going into this detail or whether we should take the stance of "this is undocumented, so if you're relying on it then untangle it for yourself" :-)

Reply:
That looks good to me, thanks!

I think it should be enough for now, but we'll likely want to go through the details a bit more once we get the pieces together and have a documented way of adding filters to the Snippets listing view.